app/javascript/css/calculator.css.scss

```diff
@@ -0,0 +1,88 @@
+.calculate-btn {
+  width: 186px;
+  height: 56px;
+  padding-top: 2px;
+  text-align: center;
+  color: #fff;
+  background-color: $success;
+  border: none;
+  border-radius: 4px;
+  font-size: 14px;
+  font-family: 'Nunito', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: 2px;
+}
+
+#calc {
+  width:24px;
+  height:32px;
+  padding-top: 2px;
+  margin-right: 10px;
+}
+
+.form-p {
+  font-size:20px;
+  line-height: 30px;
+  font-weight: 400;
+  color: #666666;
+  position: relative;
+  top: 48px;
+  left: 56px;
+}
+
+.src {
+  position: relative;
+  top: 30px;
+  margin-left: 20px;
+}
+
+.form {
+  padding-top: 15px;
+  padding-left: 55px;
+  width: 800px;
+  height:360px;
+  text-align: start;
+  display: flex;
+}
+
+.flex-item {
+  margin-top: 40px;
+  width: fit-content;
+}
+
+.flex-item input[type="text"], .flex-item input[type="date"]{
+  height: 50px;
+  margin-right: 20px;
+}
+
+.flex-item input[type="text"]:focus, .flex-item input[type="date"]:focus {
+  outline:none;
+}
+
+#baby_weight {
+  width: 160px;
+  padding-left: 20px;
+}
+
+#diapers_per_day {
+  width: 140px;
+  padding-left: 10px;
+}
+
+#date {
+  width:236px;
+  padding-left: 20px;
+  padding-right: 15px;
+  color: #999999;
+}
+
+.result-btn {
+  width:250px;
+  text-align: center;
+}
+
+.result-btn #calc {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
Configure your editor to add a new line at the end of the file, please.

Reply:
![done](https://user-images.githubusercontent.com/77724437/160792133-14edfa16-fb17-49f2-9591-376e74e787f4.png)
Done